docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,171 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you use the
+`openstack` CLI utility.
+
+## Prerequisites
+
+Although not strictly necessary, it is certainly helpful to have some
+familiarity with [creating servers in {{brand}}](new-server.md),
+especially with the OpenStack CLI.
+
+## Instantiating a vGPU server
+
+Before you begin, please make sure to source [your `openstack` RC file
+first](../../getting-started/enable-openstack-cli.md).
+
+As we discussed in the [guide on creating
+servers](/howto/openstack/nova/new-server), an `openstack` command for
+instantiating a new server looks like this:
+
+```console
+openstack server create \
+    --flavor $FLAVOR_ID \
+    --image $IMAGE_NAME \
+    --boot-from-volume $VOL_SIZE \
+    --network $NETWORK_NAME \
+    --security-group $SEC_GROUP_NAME \
+    --key-name $KEY_NAME \
+    --wait \
+    $SERVER_NAME
+```
+
+For servers with vGPU support, the only acceptable value for variable
+`IMAGE_NAME` is `Ubuntu 22.04 NVGRID x86_64`. Additionally, the value
```

Comment:
I would suggest you use non-breaking spaces in the image name here:
```suggestion
`IMAGE_NAME` is `Ubuntu 22.04 NVGRID x86_64`. Additionally, the value
```
(I guess this doesn't look to clear in the review web UI, hence the explanation.)

Reply:
Sorry, but I cannot type a non-breaking space in Mac OS :/ Pressing the Option key together with the Space key is supposed to do the trick, but that doesn't seem to work.